python-lib/dku_stress_test_center/model_accessor.py

```diff
@@ -44,5 +51,6 @@ def get_per_feature(self):
     def get_predictor(self):
         return self.model_handler.get_predictor()
 
-    def predict(self, df):
-        return self.get_predictor().predict(df)
+    def predict_and_concatenate(self, df, dropna=True):
```

Comment:
I don't know why we would have nan predictions and why we need the dropna here. When we drop the rows here, are we sure to drop them in the df then?

Reply:
1. We can have empty predictions if one of the features uses `Drop rows (do not predict them either)` as it missing handling method, hence we need the `.dropna(subset=[DkuStressTestCenterConstants.PREDICTION])` on line 55.

2. I am not so sure however why I added the `dropna(how='all')` below 😅 I'll try to see if it can safely be removed

Reply:
Done in 408cce0